src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdkAppEngineDevServer1.java

```diff
@@ -145,4 +161,33 @@ public void stop(StopConfiguration configuration) throws AppEngineException {
       }
     }
   }
+
+  @VisibleForTesting
+  void checkAndWarnIgnored(Object valueToIgnore, String propertyName) {
```

Comment:
I think the property name is just a "hint" here for the error message. The subject of the method is valueToIgnore. I mean I'd like to keep it like this for now, but open to discussing it.